### PR TITLE
Add Squeak (Smalltalk) notes

### PR DESCRIPTION
**Pull request by cstes**

### CODE REVIEW

**Comment by cstes** [comment]
I'm ok with "not documenting" the 'pfexec squeak' (to run squeak in a profile that gives it privilege ).  However I would like to keep the section on how to install OpenIndiana in this document.   From  a Squeak user perspective, those notes could be helpful on how to install the OpenIndiana operating system so that it is suitable for Squeak : as an example I discuss the "text based installer" followed by the "install MATE desktop" .    This is not Squeak specific, but a Squeak user who wants to test OpenIndiana may find it useful to see how I installed my OpenIndiana test server, so that it is documented how the O.S. was set up.

**Comment by cstes** [comment]
I removed the stuff on : pfexec -P proc_priocntl squeak Cuis5.0-4112.image

this is to give the proc_priocntl privilege to the squeak process.   It is not essential, so I'm fine with not documenting this.

**Comment by cstes** [comment]
I added a note "OpenIndiana for Squeak Users" since in some sense, I think the way I see this blog, is that it documents some OpenIndiana concepts (installation, mediator etc.) for Squeak users.   The target public is in my opinion potential Squeak users - there may be some (very few) who may be interested in OpenIndiana.

**Comment by cstes** [comment]
I've changed some stuff,  for me this documentation builds.   I get a warning that the 'squeak.md' is not referenced from the nav: in the mkdocs.yml  but I'm not sure what you actually want, my understanding is that you want the squeak documentation to be referenced from the community.md and not from the mkdocs.yml.

**Comment by cstes** [comment]
I've added some "pkg contents -r -t depend squeak-4" examples to check the required dependencies for the packages, so that users can verify this, before they install squeak.   However after adding this, the "checks fail".

I'm not sure why.

 $ /usr/ruby/2.6/bin/mdl --version
0.9.0

for me the "mdl" (markdown lint) step seems to work:

 $ /usr/ruby/2.6/bin/mdl -s markdownlint-rules.rb .

this works for me.

**Comment by cstes** [comment]
Any news on this oi-docs issue please ??

These documentation notes explain the OpenIndiana specifics to potential Squeak users. 

I think specifically the "squeak mediator" stuff is interesting as there exist many flavors of Squeak  (stack,cog,sista,newspeak,pharo,hydra whatever) and this is really a nice feature of IPS that you have mediator with different implementations of the same package.

Specifically to compare when some of the VMs have 'odd' behavior it is important to be able to compare for example the old stack implementation with the newer 'cog' implementation.

I am preparing a new version 5.0.8 of squeak-5 and squeak-5c but the documentation will not change.

File: docs/handbook/community/squeak.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+Addison-Wesley Publishing Company, Reading,
+Massachussetts, Menlo Park, California, 1983, ISBN 0-201-11371-6
+
+There also exist more recent books on Squeak.  See the <a href="http://www.squeak.org" target="_blank">Squeak website</a> for more information, specifically the documentation section.  Also see the Squeak Wiki for more information : <a href= "http://wiki.squeak.org" target="_blank">Squeak Wiki</a>
+
+## How to install Squeak Smalltalk-80 on OpenIndiana
+
+<i class="fa fa-info-circle fa-lg" aria-hidden="true"></i>OpenIndiana Prerequisites
+<div class="well">
+<p>When installing OpenIndiana, you will need a desktop (graphical) environment, such as the MATE desktop.
```
Comment (pyhalov) [inline docs/handbook/community/squeak.md:36]: When installing Squeak on OI?

**Comment by cstes** [comment]
any news please ?

**Comment by cstes** [comment]
I've added some notes on pkg update-ing squeak-5 as this is not so simple it seems, when squeak is in the userland-incorporation  (using pkg facet to version-lock set to false).   These notes may help squeak users that are unaware of these aspects of OpenIndiana ...   The idea of this documentation is to document some useful tricks to Squeak users on how to use OpenIndiana specifically in the context of squeak.

File: docs/handbook/community/squeak.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+        # pkg install -v --accept mate_install
+
+        # svcadm enable graphical-login/lightdm
+
+This enables the "light display manager".
+
+In any case, even if you do a direct desktop installation (without running through the text based installation of OpenIndiana), take care about the ip address and hostname of the system.
+
+Squeak will try to use the ip address that corresponds with the hostname of the system as its ip address.
```
Comment (pyhalov) [inline docs/handbook/community/squeak.md:48]: corresponds to

**Comment by cstes** [comment]
Hi, this was approved but as far as I can see the notes were never happened ...

**Comment by cstes** [comment]
Hi, this was approved but as far as I can see the notes were never happened ...

**Comment by cstes** [comment]
Sorry I meant to say : this was approved but the notes were not added.

**Pull request merged**